eva_submission/steps/detect_contigs_naming_convention.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2022 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import gzip
+import os.path
+from argparse import ArgumentParser
+from collections import defaultdict
+
+import requests
+import yaml
+from cached_property import cached_property
+from ebi_eva_common_pyutils.logger import AppLogger
+from retry import retry
+
+# Order in which the naming convention will be kept if multiple are equivalent
+naming_convention_priority = {
+    'enaSequenceName': 1,
+    'genbankSequenceName': 2,
+    'ucscName': 3,
+    'insdcAccession': 4,
+    'refseq': 5
+}
+
+
+class ContigAliasClient:
```

Comment:
Can these methods be added to the client in [pyutils](https://github.com/EBIvariation/eva-common-pyutils/blob/master/ebi_eva_common_pyutils/contig_alias/contig_alias.py)?